R/tm_g_bivariate.R

```diff
@@ -327,16 +327,17 @@ ui_g_bivariate <- function(id, ...) {
         )
       )
     ),
-    forms = teal::get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
```

Comment:
### Explanation comment
formerly `teal::get_rcode_ui` (and `teal::get_rcode_srv`) is dependent on chunks, thus it needs to be replaced.